OLD_setup.py

```diff
@@ -176,8 +176,8 @@ def install_scs(**kwargs):
     extra_compile_args = ["-O3"]
     extra_link_args = []
     libraries = []
-    sources = ["src/scspy.c"] + glob("scs/src/*.c") + glob("scs/linsys/*.c")
-    include_dirs = ["scs/include", "scs/linsys"]
+    sources = ["scs/scspy.c"] + glob("scs_source/src/*.c") + glob("scs_source/linsys/*.c")
+    include_dirs = ["scs_source/include", "scs_source/linsys"]
```

Comment:
Do we have to rename the directories? It would be nice to keep them as before if possible.

Reply:
I'm not sure if there was a reason for this that triggered the change, but Meson should be able to handle any layout in principle (at least any that worked in `numpy.distutils` I'd think).

Reply:
It wasn't standard; package should be in directory named after it. (You want standardization for ease of maintenance.) Go on and rename them back if you prefer.